pkgs/applications/misc/organicmaps/default.nix

```diff
@@ -0,0 +1,94 @@
+{ lib
+, mkDerivation
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, cmake
+, ninja
+, pkg-config
+, which
+, python3
+, makeWrapper
+, qtbase
+, qtsvg
+, libGLU
+, zlib
+}:
+
+mkDerivation rec {
+  pname = "organicmaps";
+  version = "2021-05-15";
+
+  src = fetchFromGitHub {
+    owner = "organicmaps";
+    repo = "organicmaps";
+    rev = version;
+    sha256 = "sha256-g5AV72JZjgcWKTBkve1dAi9W9ocdeWAROy+HCrsILLE=";
+    fetchSubmodules = true;
+  };
+
+  # Update desktop file, remove on next release if this pr is merged:
+  # https://github.com/organicmaps/organicmaps/pull/418
+  patches = [(fetchpatch {
+    url = "https://github.com/organicmaps/organicmaps/pull/418/commits/0bbd1e2af3a6c4af7a703ab23db7e265a50af342.patch";
+    sha256 = "sha256-7khIf1JFNcwTMA7T9EdHz+CyHjRyUommLtTo6q/b8U8=";
+  })];
+
+  # Disable certificate check. It's dependent on time
+  postPatch = ''
+    echo "exit 0" > tools/unix/check_cert.sh
+    # And for some reason the patchPhase doesn't apply the renaming:
+    mv qt/res/MapsWithMe.desktop qt/res/OrganicMaps.desktop
+  '';
+
+  nativeBuildInputs = [
+    cmake
+    ninja
+    pkg-config
+    which
+    python3
+    makeWrapper
+  ];
+
+  # Most dependencies are vendored
+  buildInputs = [
+    qtbase
+    qtsvg
+    libGLU
+    zlib
+  ];
+
+  # Yes, this is PRE configure. The configure phase uses cmake
+  preConfigure = ''
+    bash ./configure.sh
```

Comment:
```suggestion
    ./configure.sh
```

Reply:
I would have to patch the shebang then. Should I do that?

Reply:
Then either way is fine.